### PR TITLE
Feature/sie56 modal

### PR DESCRIPTION
**Pull request by harmanwho**

### CODE REVIEW

File: src/stories/components/Modal/Modal.js [hunk 1]
```diff
@@ -0,0 +1,84 @@
...
+        isButton={true}/>
+      <div
+        hidden={!modalOpen}
+        id='modalPop'
+        className={['modal', `modal--${theme}`].join(' ').trim()}
+      >
+        <div
+          className={
+            (modalOpen) ?
+              'modal--content modal--open' :
```
Comment (alecherryy) [inline src/stories/components/Modal/Modal.js:50]: @harmanwho classes for child elements should be `modal__el`; replace this with `modal__content`.

File: src/stories/components/Modal/Modal.js [hunk 2]
```diff
@@ -0,0 +1,84 @@
...
+              'modal--content'}
+        >
+          <span
+            onClick={toggleModal}
+            className='modal--content--close'
+          />
+          {children}
+        </div>
```
Comment (alecherryy) [inline src/stories/components/Modal/Modal.js:58]: Semantically speaking this should be a button. You can have text inside of it, visually hide it via CSS and use the pseudo-element to include the icon. This is the code snippet I use:
```
display: block;
color: transparent;
font: 0/0 a;
height: 24px;
text-shadow: none;
width: 24px;
position: relative;
```
It visually hides the text, but keeps it accessible to screen readers. It's pretty neat! Let me know if you have any questions!

File: src/stories/components/Modal/styles.scss [hunk 3]
```diff
@@ -0,0 +1,50 @@
...
+/* The Modal (background) */
+.modal {
+  position: fixed;
+  z-index: 1;
+  left: 0;
+  top: 0;
+  width: 100%;
+  height: 100%;
+  overflow: auto;
+  // background-color: rgba(0,0,0,0.4);
```
Comment (alecherryy) [inline src/stories/components/Modal/styles.scss:15]: You can un-comment this and use this RGB value: `rgba(255, 255, 255, 0.9)`

**Review by alecherryy — CHANGES_REQUESTED** [review]

File: src/stories/components/Modal/Modal.js (hunk 2 above)
Comment (harmanwho) [reply src/stories/components/Modal/Modal.js:58]: @alecherryy That's a great idea, I did not know that!
I tried it but I wasn't able to hide the button; an empty block of sorts still appears.
Screen Shot 2021-03-03 at 2 23 22 PM

**Review by alecherryy — APPROVED** [review]

File: src/stories/components/Modal/Modal.js [hunk 4]
```diff
@@ -0,0 +1,88 @@
...
+        <div
+          className={
+            (modalOpen) ?
+              'modal__content modal--open' :
+              'modal__content'}
+        >
+          {/* <span
+            onClick={toggleModal}
+            className='modal__content--close'
+          /> */}
```
[the remark this answers is not in the document]
Comment (harmanwho) [reply src/stories/components/Modal/Modal.js]: No, I didn't. Sorry

**Pull request merged**